pkg/chaosctl/common/common.go

```diff
@@ -23,6 +23,8 @@ import (
 	"strings"
 	"time"
 
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
```

Comment:
need to format this import

Reply:
@STRRL PTAL

Reply:
Fixed manually. A little weird, I can't reformat these imports by `make check`.